src/components/home/chat-bubble/components/__tests__/date-indicator.test.tsx

```
@@ -0,0 +1,110 @@
+    conversationId: "conv1",
+    createdAt: new Date().toISOString(),
+    updatedAt: new Date().toISOString(),
+    _creationTime: 1234567890,
+  };
+
+  beforeEach(() => {
+    vi.clearAllMocks();
+  });
+
+  it("renders date indicator when no previous message", () => {
+    (isSameDay as any).mockReturnValue(false);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)